### PR TITLE
documentation updates for 0.5.1 release

### PR DESCRIPTION
**Pull request by ryane**

### CODE REVIEW

File: docs/getting_started/index.rst [hunk 1]
```diff
@@ -60,20 +60,32 @@ Deploying software via Ansible
...
 .. code-block:: shell
 
-  ansible all -i plugins/inventory/terraform.py -m ping 
+  ansible all -m ping
 
 If any servers fail to connect, check your connection by adding ``-vvvv``
 for verbose SSH debugging and try again to view the errors in more detail.
 
-Next, deploy the software. First, you'll need to customize a playbook. A sample
+Next, it is important that we upgrade the operating system packages on all
```
Comment (BrianHicks) [inline docs/getting_started/index.rst]: "it is important" understates it a little. Maybe put something in a warning block about what'll happen if you *don't* upgrade first?

Comment (ChrisAubuchon) [inline docs/getting_started/index.rst]: Lol. As of right now, either way Consul is broken...

**Comment by ryane** [comment]
@BrianHicks does the latest two commits cover it? @ChrisAubuchon I was planning to add the remaining consul failure to the known issues release notes. In my experience, it is usually recoverable.

**Comment by BrianHicks** [comment]
@ryane works for me. Love the commit "big warning" commit message. :P

**Pull request merged**